ansible/roles/avocado/tasks/main.yaml

```diff
@@ -0,0 +1,79 @@
+- name: Setup avocado repository
+  copy:
+    src: avocado.repo
+    dest: /etc/yum.repos.d/avocado.repo
+    owner: root
+    group: root
+    mode: 0644
+  tags:
+    - setup
+
+- name: Install packages
+  package: name={{item}}
+  notify: vt-bootstrap
+  with_items:
+    - avocado
+    - avocado-plugins-vt
+    - libvirt
+    - qemu-kvm
+    - wget
+  tags:
+    - setup
```

Comment:
I have used the `pkg_install` role for this in the baremetal setup, Could you take a look and consider using it as well? The main difference is that the list of deps is moved to defaults/main.yaml and the `pkg_install` goes into the `dependencies` field of this role. See:

The pkg_install role containing the boilerplate for installing packages:
https://github.com/open-power-host-os/infrastructure/blob/master/ansible/roles/pkg_install/tasks/main.yaml

And its usage:

The listing of the package dependencies that a role needs installed:
https://github.com/open-power-host-os/infrastructure/blob/master/ansible/roles/baremetal-ctrl/defaults/main.yaml#L5

The role pulling in `pkg_install` to do the work:
https://github.com/open-power-host-os/infrastructure/blob/master/ansible/roles/baremetal-ctrl/meta/main.yaml#L4